lms/templates/courseware/dates.html

```diff
@@ -36,7 +38,7 @@ <h2 class="hd hd-2 date-title">
                                 <div class="timeline-date-content">
                                         % if block.date:
                                             <div class="timeline-date">
-                                                ${block.date.strftime(block.date_format)}
+                                                ${block.date.astimezone(pytz.timezone(user_timezone)).strftime(block.date_format)}
```

Comment:
Do we do this calculation on the backend in other places? Or do we use javascript to adjust in the browser?

Reply:
Going to look into other options for this as I'll also need to handle the translations as well, this definitely has been solved elsewhere.